app_dart/lib/src/model/appengine/task.dart

```diff
@@ -181,6 +182,56 @@ class Task extends Model<int> {
     return task;
   }
 
+  /// Creates a [Task] based on a buildbucket [bb.Build].
+  static Future<Task> fromBuildbucketV2Build(
```

Comment:
Let's add a test to cover the logic here.

Reply:
Done.